app.js

```diff
@@ -0,0 +1,4 @@
+import { config } from './config.js';
+import Server from './index.js';
+
+const server = new Server(config.port);
```

Comment:
We can skip passing `config.port` (I mean we can create an instance without any arguments) because we will use `config.port` by default